examples/stable_diffusion_xl/gm/modules/diffusionmodules/openaimodel.py

```diff
@@ -154,16 +153,11 @@ def __init__(
         self.use_scale_shift_norm = use_scale_shift_norm
         self.exchange_temb_dims = exchange_temb_dims
 
-        if isinstance(kernel_size, Iterable):
-            padding = [k // 2 for k in kernel_size]
-        else:
-            padding = kernel_size // 2
-
         self.in_layers = nn.SequentialCell(
             [
-                normalization(channels),
+                Normalization(channels),
                 nn.SiLU(),
-                conv_nd(dims, channels, self.out_channels, kernel_size, padding=padding, pad_mode="pad"),
+                conv_nd(dims, channels, self.out_channels, kernel_size, pad_mode="same"),
```

Comment:
this implementation reduces the functionality it has before, may be better to keep it as original, sames for other modificaitons on `conv_nd`